xcell/mappers/mapper_HSC_DR1wl.py

```diff
@@ -0,0 +1,278 @@
+from .mapper_base import MapperBase
+from .utils import get_map_from_points
+from astropy.table import Table, vstack
+import os
+import numpy as np
+import healpy as hp
+
+
+class MapperHSCDR1wl(MapperBase):
```

Comment:
Could you add some documentation? At least the arguments to pass to the mapper and its default paths/values

Reply:
Added docs for the different arguments. Better docs are needed for all mappers (and this is already in #73 )

Reply:
Could you add the default values in the docstring? E.g. `'depth_cut' : i-band magnitude cut (default 24.5)` or sth like that

Reply:
Done (BTW, if you mark comments as resolved I may not see them)